<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-E2PSIL5I" failures="1" tests="1" name="UserAccountAndListsTest" time="231.315" errors="0" timestamp="2023-02-17T22:47:42 IST">
  <testcase classname="amazon.UserAccountAndLists.TestScripts.LoginToApplicationTest" name="loginIntoAppTest" time="2.319">
    <failure type="java.lang.AssertionError" message="[ASSERTION FAILED]: Sign in unsuccessful expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: [ASSERTION FAILED]: Sign in unsuccessful expected [true] but found [false]
at amazon.UserAccountAndLists.TestScripts.LoginToApplicationTest.loginIntoAppTest(LoginToApplicationTest.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- loginIntoAppTest -->
</testsuite> <!-- UserAccountAndListsTest -->
